app/components/header_component/view.rb

```diff
@@ -2,14 +2,15 @@
 
 module HeaderComponent
   class View < ViewComponent::Base
-    attr_accessor :is_signed_in, :user_name, :user_profile_link, :signout_link, :list_of_users_path, :hosting_environment
+    attr_accessor :is_signed_in, :user_name, :user_profile_link, :signout_link, :list_of_users_path, :mou_path, :hosting_environment
 
-    def initialize(is_signed_in:, user_name:, user_profile_link:, signout_link:, list_of_users_path:, hosting_environment: HostingEnvironment)
+    def initialize(is_signed_in:, user_name:, user_profile_link:, signout_link:, list_of_users_path:, mou_path:, hosting_environment: HostingEnvironment)
```

Comment:
Is there anyway we could condense this?

if a method has a lot of params like this its usually a sign of a code smell.

Reply:
yeah - I guess creating a HeaderComponentOptions object would be the answer?

Reply:
yeah potentially a service which returns these options would be better than trying to replicate and update all calls to the component everytime we add another param.

Reply:
I've refactored the header component so that links are passed in